### DISCUSSION TITLE
Copilot does not work on vscode, output shows CodeQuote disabled.

### DISCUSSION THREAD

**Discussion by Ken0507** [community/community#50464] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Copilot is not working in my vscode, I tried two computers (all Macs) and it does not work at all. The log gives the following info:
[INFO] [codequote] [2023-03-20T02:14:30.700Z] CodeQuote disabled

**Comment by iamcjwilliams** [comment]
@Ken0507 Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot

**Comment by jealcalat** [comment]
Same bug. When downgrade to the previous version the message disappears. Upgrade returns the bug.

**Comment by rajhlinux** [comment]
I get this error:
"[INFO] [codequote] [2023-03-27T01:36:34.272Z] CodeQuote disabled"

When I use previous version, nothing works.

When I go to a much further previous version it throw error that I need to update VS code.

Using FreeBSD 13.1

**Comment by mdguo** [comment]
downgrading to 1.79 worked for me
